src/test/groovy/graphql/parser/MultiSourceReaderTest.groovy

```diff
@@ -10,6 +13,8 @@ class MultiSourceReaderTest extends Specification {
         multiSource.close()
     }
 
+    // Does not work on Windows
+    @IgnoreIf({ PreconditionContext it -> it.os.windows })
```

Comment:
Is there any issue behind this one, or an explanation why this does not work on Windows?

Reply:
#2709 (test below is the root cause)